test/e2e/network_test.go

```diff
@@ -552,11 +552,10 @@ var _ = Describe("Podman network", func() {
 
 	It("podman network create/remove macvlan", func() {
 		net := "macvlan" + stringid.GenerateRandomID()
-		nc := podmanTest.Podman([]string{"network", "create", "--macvlan", "lo", net})
+		nc := podmanTest.Podman([]string{"network", "create", "--driver", "macvlan", "--opt", "parent=lo", net})
```

Comment:
we already test that basically on line 590 and below so I would suggest to remove this test case completely to safe a few ms of test time.